account_cutoff_prepaid/tests/test_account_cutoff_prepaid.py

```diff
@@ -1,82 +1,78 @@
         super(TestCutoffPrepaid, self).setUp()
-        self.inv_model = self.registry('account.invoice')
-        self.cutoff_model = self.registry('account.cutoff')
+        self.inv_model = self.env['account.invoice']
+        self.cutoff_model = self.env['account.cutoff']
+        self.account_model = self.env['account.account']
+        self.journal_model = self.env['account.journal']
+        self.account_expense = self.account_model.search([(
+            'user_type_id',
+            '=',
+            self.env.ref('account.data_account_type_expenses').id)], limit=1)
+        self.account_payable = self.account_model.search([(
+            'user_type_id',
+            '=',
+            self.env.ref('account.data_account_type_payable').id)], limit=1)
+        self.account_cutoff = self.account_model.search([(
+            'user_type_id',
+            '=',
+            self.env.ref('account.data_account_type_current_liabilities').id)],
+            limit=1)
+        self.cutoff_journal = self.journal_model.search([], limit=1)
+        self.purchase_journal = self.journal_model.search([(
+            'type', '=', 'purchase')], limit=1)
 
     def _date(self, date):
         """ convert MM-DD to current year date YYYY-MM-DD """
         return time.strftime('%Y-' + date)
 
     def _days(self, start_date, end_date):
-        start_date = parse_date(self._date(start_date))
-        end_date = parse_date(self._date(end_date))
+        start_date = fields.Date.from_string(self._date(start_date))
+        end_date = fields.Date.from_string(self._date(end_date))
         return (end_date - start_date).days + 1
 
     def _create_invoice(self, date, amount, start_date, end_date):
-        inv_id = self.inv_model.create(self.cr, self.uid, {
-            'journal_id': self.ref('account.expenses_journal'),
+        invoice = self.inv_model.create({
             'date_invoice': self._date(date),
-            'account_id': self.ref('account.a_recv'),
-            'partner_id': self.ref('base.res_partner_17'),
+            'account_id': self.account_payable.id,
+            'partner_id': self.env.ref('base.res_partner_2').id,
+            'journal_id': self.purchase_journal.id,
             'type': 'in_invoice',
-            'invoice_line': [(0, 0, {
+            'invoice_line_ids': [(0, 0, {
                 'name': 'expense',
                 'price_unit': amount,
                 'quantity': 1,
-                'account_id': self.ref('account.a_expense'),
+                'account_id': self.account_expense.id,
                 'start_date': self._date(start_date),
                 'end_date': self._date(end_date),
             })],
         })
-        workflow.trg_validate(self.uid, 'account.invoice', inv_id,
-                              'invoice_open', self.cr)
-        inv = self.inv_model.browse(self.cr, self.uid, inv_id)
-        self.assertEqual(amount, inv.amount_untaxed)
-        return inv_id
+        invoice.signal_workflow('invoice_open')
```

Comment:
The workflow has been removed, you must use `invoice.action_invoice_open()` now.